javascript/JavaScriptLexer.g4

```diff
@@ -33,6 +34,7 @@ channels { ERROR }
 
 options { superClass=JavaScriptBaseLexer; }
 
+HashBangLine:                   '#!' ~[\r\n\u2028\u2029]*; // only allowed at start
```

Comment:
Have you seen how it is handled in PHP grammar? https://github.com/antlr/grammars-v4/blob/master/php/PhpLexer.g4#L46